packages/mobile/src/backup/utils.ts

```diff
@@ -14,11 +14,13 @@ export function selectQuizWordOptions(correctWord: string, allWords: string[], n
   const wordOptions = []
   const correctWordPosition = Math.floor(Math.random() * numOptions)
   for (let i = 0; i < numOptions; i++) {
-    wordOptions.push(
-      i === correctWordPosition
-        ? correctWord
-        : allWords[Math.floor(Math.random() * allWords.length)]
-    )
+    if (i === correctWordPosition) {
+      wordOptions.push(correctWord)
+      continue
+    }
+    const randomWord = allWords[Math.floor(Math.random() * allWords.length)]
+    wordOptions.push(randomWord)
+    allWords = allWords.filter((w) => w !== randomWord)
```

Comment:
Nit: I think slicing out the index of the chosen word would be quicker than filtering the whole list